### PR TITLE
Reformatting Featurization of Transform and Misc files in Transform to Width 85

### PR DESCRIPTION
**Pull request by RadicalRayan**
Guidelines followed:
-85 characters per line
-Use 4 spaces for indentation
-Dot and open parentheses stay on same line as function
-If not a preexisting line under line that we break, add an extra line after it
-Don't indent comments
-Don't break a comment if it represents output
-Don't break links
-If applicable, break right before $
-Keep math op together

Fix for issue #3478
*Misc. refers to those files found in the Transform directory but not in another subdirectory

### CODE REVIEW

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeSupervisedBinning.cs [hunk 1]
```diff
@@ -12,74 +12,120 @@ public class NormalizeSupervisedBinning
...
-
-            // NormalizeSupervisedBinning normalizes the data by constructing bins based on correlation with the label column and produce output based on 
-            // to which bin original value belong but make sure zero values would remain zero after normalization.
-            // Helps preserve sparsity.
-            var normalizeFixZero = mlContext.Transforms.NormalizeSupervisedBinning("Features", labelColumnName: "Bin", mininimumExamplesPerBin: 1, fixZero: true);
-
-            // Now we can transform the data and look at the output to confirm the behavior of the estimator.
-            // This operation doesn't actually evaluate data until we read the data below.
+            data = mlContext.Transforms.Conversion.MapValueToKey("Bin").Fit(data)
+                .Transform(data);
```
Comment (sierralee51) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/NormalizeSupervisedBinning.cs]: should there be an extra line afterwards?

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TreeFeaturization/BinaryClassificationFeaturization.ttinclude [hunk 2]
```diff
@@ -39,39 +43,52 @@ namespace Samples.Dynamic.Transforms.TreeFeaturization
...
+                    "different tree-based feature vectors:");
+
+                Console.WriteLine("  Trees' output values: [" + String.Join(",",
+                    transformedDataPoint.Trees) + "].");
+
+                Console.WriteLine("  Leave IDs' 0-1 representation: [" + String
+                    .Join(",", transformedDataPoint.Leaves) + "].");
+
+                Console.WriteLine("  Paths IDs' 0-1 representation: [" + String
+                    .Join(",", transformedDataPoint.Paths) + "].");
```
Comment (sierralee51) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TreeFeaturization/BinaryClassificationFeaturization.ttinclude]: should there be a blank line after this?

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TreeFeaturization/BinaryClassificationFeaturization.ttinclude [hunk 3]
```diff
@@ -82,13 +99,16 @@ namespace Samples.Dynamic.Transforms.TreeFeaturization
                 {
                     Label = label,
                     // Create random features that are correlated with the label.
-                    // For data points with false label, the feature values are slightly increased by adding a constant.
-                    Features = Enumerable.Repeat(label, 3).Select(x => x ? randomFloat() : randomFloat() + <#=DataSepValue#>).ToArray()
+                    // For data points with false label, the feature values are
+                    // slightly increased by adding a constant.
+                    Features = Enumerable.Repeat(label, 3).Select(x => x ?
+                    randomFloat() : randomFloat() + <#=DataSepValue#>).ToArray()
```
Comment (sierralee51) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/TreeFeaturization/BinaryClassificationFeaturization.ttinclude]: indent?

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContributionCalibrated.cs [hunk 4]
```diff
@@ -19,7 +20,8 @@ public static void Example()
             // Convert training data to IDataView.
             var data = mlContext.Data.LoadFromEnumerable(samples);
 
-            // Create a pipeline to concatenate the features into a feature vector and normalize it.
+            // Create a pipeline to concatenate the features into a feature vector
+            // and normalize it.
             var transformPipeline = mlContext.Transforms.Concatenate("Features", 
                     new string[] { nameof(Data.Feature1), nameof(Data.Feature2) })
                 .Append(mlContext.Transforms.NormalizeMeanVariance("Features"));
```
Comment (sayanshaw24) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContributionCalibrated.cs]: Extra spaces on line 26

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContributionCalibrated.cs [hunk 5]
```diff
@@ -114,7 +133,9 @@ private class ScoredData : Data
                 };
 
                 // Create a Boolean label with noise.
-                var value = bias + weight1 * data.Feature1 + weight2 * data.Feature2 + rng.NextDouble() - 0.5;
+                var value = bias + weight1 * data.Feature1 + weight2 * data.Feature2
+                    + rng.NextDouble() - 0.5;
+
```
Comment (sayanshaw24) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CalculateFeatureContributionCalibrated.cs]: Maybe move the + symbol to line 136

File: docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Concatenate.cs [hunk 6]
```diff
@@ -9,38 +9,56 @@ public static class Concatenate
...
-            // The Concatenate requires all columns to be of same type.
-            var pipeline = mlContext.Transforms.Conversion.ConvertType("Feature3", outputKind: DataKind.Single)
-                .Append(mlContext.Transforms.Concatenate("Features", new[] { "Feature1", "Feature2", "Feature3" }));
+            // Please note that the "Feature3" column is converted from int32 to
+            // float using the ConvertType. The Concatenate requires all columns to
+            // be of same type.
+            var pipeline = mlContext.Transforms.Conversion.ConvertType("Feature3",
+                outputKind: DataKind.Single)
+                .Append(mlContext.Transforms.Concatenate("Features", new[]
+                    { "Feature1", "Feature2", "Feature3" }));
```
Comment (sayanshaw24) [inline docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Concatenate.cs]: Extra spaces on line 52

Comment (RadicalRayan) [reply docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Concatenate.cs]: intentional to highlight parameter of the append method

**Pull request merged**